server/src/main/java/org/apache/cassandra/sidecar/datahub/SchemaReporter.java

```diff
@@ -96,45 +110,73 @@ public SchemaReporter(@NotNull IdentifiersProvider identifiersProvider,
      * @param keyspaceConverters a {@link List} of {@link KeyspaceToAspectConverter} instances to use
      * @param tableConverters a {@link List} of {@link TableToAspectConverter} instances to use
      * @param emitterFactory an instance of {@link EmitterFactory} to use
+     * @param reportingMetrics an instance of {@link SchemaReportingMetrics} to use
      */
     protected SchemaReporter(@NotNull IdentifiersProvider identifiersProvider,
                              @NotNull List<ClusterToAspectConverter<? extends RecordTemplate>> clusterConverters,
                              @NotNull List<KeyspaceToAspectConverter<? extends RecordTemplate>> keyspaceConverters,
                              @NotNull List<TableToAspectConverter<? extends RecordTemplate>> tableConverters,
-                             @NotNull EmitterFactory emitterFactory)
+                             @NotNull EmitterFactory emitterFactory,
+                             @NotNull SchemaReportingMetrics reportingMetrics)
     {
         this.identifiersProvider = identifiersProvider;
         this.clusterConverters = clusterConverters;
         this.keyspaceConverters = keyspaceConverters;
         this.tableConverters = tableConverters;
         this.emitterFactory = emitterFactory;
+        this.reportingMetrics = reportingMetrics;
     }
 
     /**
-     * Public method for converting and reporting the Cassandra schema
+     * Public method for converting and reporting the Cassandra schema when triggered by a scheduled periodic task
      *
      * @param cluster the {@link Cluster} to extract Cassandra schema from
      */
-    public void process(@NotNull Cluster cluster)
+    public void processScheduled(@NotNull Cluster cluster)
     {
-        process(cluster.getMetadata());
+        process(cluster.getMetadata(), reportingMetrics.startedSchedule.metric);
     }
 
     /**
-     * Public method for converting and reporting the Cassandra schema
+     * Public method for converting and reporting the Cassandra schema when triggered by a received API request
      *
      * @param metadata the {@link Metadata} to extract Cassandra schema from
      */
-    public void process(@NotNull Metadata metadata)
+    public void processRequested(@NotNull Metadata metadata)
     {
+        process(metadata, reportingMetrics.startedRequest.metric);
+    }
+
+    /**
+     * Private method for converting and reporting the Cassandra schema
+     *
+     * @param metadata the {@link Metadata} to extract Cassandra schema from
+     * @param started the {@link DeltaGauge} for the metric counting invocations
+     */
+    private void process(@NotNull Metadata metadata,
+                         @NotNull DeltaGauge started)
+    {
+        String action = " reporting schema for cluster with identifiers " + identifiersProvider;
+        LOGGER.info("Started" + action);
```

Comment:
Use the value placeholder of logger to eval lazily. Update the other places too. 

```suggestion
    LOGGER.info("Started {}", action);
```

Reply:
+1

Reply:
This did not look good and resulted in bad log messages, fixed.